### PR TITLE
Split print composer chapter into many pages

### PR DESCRIPTION
**Pull request by DelazJ**
fixes partly #543
Reading documentation about Print Composer is not so easy; It's too long
and scrolling in that page is neither easy.  This PR :
- splits this chapter into many sections : it helps to quickly identify
which sections are less documented (description of menus) or need screenshots (Items tab, Composition tab..). Writing documentation on this chapter should be easier...
- reorganises the sections among them
- adds some texts
- fixes some typo errors or outdated info (but not all)

### CODE REVIEW

**Comment by DelazJ** [comment]
As this is the first time I really work with .rst files, if somebody can review... before I begin spliting vector dialog properties chapter

File: source/docs/user_manual/print_composer/composer_items/composer_arrow.rst [hunk 1]
```diff
@@ -0,0 +1,59 @@
+|updatedisclaimer|
+
+.. _arrow_item:
+
+The Arrow item
+--------------
```
Comment (yjacolin) [inline source/docs/user_manual/print_composer/composer_items/composer_arrow.rst]: in rst file, the first time a symbol is shown for a title will be a h1 header. So it seems better to use the same symbol each time for the first header and so split the chapter in logical section (more comment in the mailing list to discuss about this)

**Comment by DelazJ** [comment]
@yjacolin I made the changes you asked for.
Unfortunately, I wrongly pushed two commits (db0e7a4 and its revert 49f8451) into the PR. Don't know how I can remove them (nor if it's even possible). Do you think it can be merged? If not, what do I need to do?

**Comment by rduivenvoorde** [comment]
@DelazJ I just build and it really looks a lot better!! Thanks for all this work.

Do not worry about some unnessecary commits, I'll just pull this in.

Again Thanks!

**Pull request merged**